src/templates/layouts/qaParagraph/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { QaParagraph as FormattedQaParagraph } from '@/types/formatted/qaParagraph'
```

Comment:
i don't think this should live in `templates/layouts/qaParagraph`. I think something like `templates/components/qa` makes more sense

`QuestionAnswer` is fine too if the shorthand qa feels funky when Group and Section are also added

Reply:
I was trying to avoid having QuestionAnswer as a layout and QuestionAnswer as a component, but if you think that would be less confusing im open to it